{"uuid": "6c3b52fc-c265-47ba-828f-2667eff4d9af", "children": ["ab9647ac-58ea-4303-ae3b-e8cd5cc237e7"], "befores": [{"name": "driver_with_two_orders", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Diplom_3\\conftest.py\", line 53, in driver_with_two_orders\n    home_page.create_new_order()\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Diplom_3\\pages\\home_page.py\", line 89, in create_new_order\n    self.wait_on_off_load_animation()\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Diplom_3\\pages\\home_page.py\", line 82, in wait_on_off_load_animation\n    self.wait_element(HomePage.loading_animation)\n  File \"C:\\Diplom_3\\pages\\base_page.py\", line 22, in wait_element\n    WebDriverWait(self.driver, 10).until(\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "open page", "status": "passed", "start": 1730191323002, "stop": 1730191323188}, {"name": "create new order", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Diplom_3\\pages\\home_page.py\", line 89, in create_new_order\n    self.wait_on_off_load_animation()\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Diplom_3\\pages\\home_page.py\", line 82, in wait_on_off_load_animation\n    self.wait_element(HomePage.loading_animation)\n  File \"C:\\Diplom_3\\pages\\base_page.py\", line 22, in wait_element\n    WebDriverWait(self.driver, 10).until(\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "add ingredient in burger", "status": "passed", "parameters": [{"name": "el_xpath", "value": "('xpath', '//img[@alt=\"Краторная булка N-200i\"]')"}], "start": 1730191323189, "stop": 1730191324312}, {"name": "wait on off load animation", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Diplom_3\\pages\\home_page.py\", line 82, in wait_on_off_load_animation\n    self.wait_element(HomePage.loading_animation)\n  File \"C:\\Diplom_3\\pages\\base_page.py\", line 22, in wait_element\n    WebDriverWait(self.driver, 10).until(\n  File \"C:\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1730191324621, "stop": 1730191334744}], "start": 1730191323188, "stop": 1730191334744}], "start": 1730191323001, "stop": 1730191334747}], "start": 1730191323001, "stop": 1730191334789}